reference/rate-limits.md

```diff
@@ -1,6 +1,14 @@
 # Rate Limits
 
-Rate limits are a powerful way to improve availability and scalability for your microservices. With Ambassador Edge Stack, individual requests can be annotated with metadata, called labels.  These labels can then be passed to a third party [rate limiting service](../services/rate-limit-service) which can then rate limit based on this data. If you do not want to write your own rate limiting service, [Ambassador Edge Stack](../../user-guide/install) includes an integrated, flexible rate limiting service.
+Rate limits are a powerful way to improve availability and scalability for your
```

Comment:
In the future, can you leave whitespace / formatting changes for a separate PR because it makes it more difficult to review? Thanks.

Reply:
Sorry! This wasn't intentional.